kubernetes/pre_post_install_task_generator.py

```diff
@@ -4,20 +4,16 @@ def create_pre_hook_manifests(source_file, target_file):
     with open (source_file) as manifest_file:
         content = manifest_file.read()
     helm_hooks = re.search(r'(?<=HOOKS:).*(?=MANIFEST)', content, re.DOTALL).group()
-    helm_pre_hooks = re.sub(r"#\s(Source)(.*)(?:attribute-service)(.*)(?:config-bootstrapper)(.+)((?:\n.+)+)\n+.+\n.+", "", helm_hooks)
-    helm_pre_hooks = re.sub(r"#\s(Source)(.*)(?:entity-service)(.*)(?:config-bootstrapper)(.+)((?:\n.+)+)\n+.+\n.+", "", helm_pre_hooks)
+    helm_pre_hooks = re.sub(r"#\s(Source)(.*)(?:config-bootstrapper)(.+)((?:\n.+)+)\n+.+\n.+", "", helm_hooks)
     with open(target_file, "w") as pre_hooks:
         pre_hooks.write(helm_pre_hooks)
 
 def create_post_hook_manifests(source_file, target_file):
     with open (source_file) as manifest_file:
         content = manifest_file.read()
-    post_attribute_hooks = re.search(r"#\s(Source)(.*)(?:attribute-service)(.*)(?:config-bootstrapper)(.+)((?:\n.+)+)\n+.+\n.+", content).group()
-    post_entity_hooks = re.search(r"#\s(Source)(.*)(?:entity-service)(.*)(?:config-bootstrapper)(.+)((?:\n.+)+)\n+.+", content).group()
+    post_install_hooks = re.search(r"#\s(Source)(.*)(?:config-bootstrapper)(.+)((?:\n.+)+)\n+.+\n", content).group()
```

Comment:
As talked, I think, we can remove this static hard-coded check, we can check `helm.sh : pre-install`, are we planning to do as follow up?

Reply:
Yup. this will be quick fix for current flow. We will do pre-install/ post-install based check as follow up.